autoload/nnn.vim

```diff
@@ -389,14 +389,16 @@ function! nnn#explorer(...) abort
 
     let l:opts.layout = l:layout
     let l:opts.ppos = { 'buf': bufnr(''), 'winid': win_getid() }
-    let l:On_exit = s:explorer_create_on_exit_callback(l:opts)
+    let l:On_exit = s:explorer_create_on_exit_callback()
 
     " create the fifo ourselves since otherwise nnn might not create it on time
     call system('mkfifo '.shellescape(s:explorer_fifo))
     let l:opts.term = s:build_window(l:layout, { 'cmd': l:cmd, 'on_exit': l:On_exit })
 
     let b:tbuf = l:opts.term
     let t:explorer_winid = l:opts.term.winhandle
+    let t:explorer_term = l:opts.term
+    let t:explorer_ppos = l:opts.ppos
```

Comment:
I'm wondering if `ppos` is relevant to explorer mode at all.
It's useful on picker since it needs to remember the previous position when switching back. While in explorer, it switch back to `#`.

Reply:
Looking at `s:explorer_create_on_exit_callback()` ppos isn't used at all. Can probably just remove it.